conformance/conformance_suite.go

```diff
@@ -106,22 +109,42 @@ func setupClients() error {
 
 		restConfig, err := clientConfig.ClientConfig()
 		if err != nil {
-			return err
+			accumulatedErrors = append(accumulatedErrors, err)
+			continue
 		}
 
 		k8sClient, err := kubernetes.NewForConfig(restConfig)
 		if err != nil {
-			return err
+			accumulatedErrors = append(accumulatedErrors, err)
+			continue
 		}
 
 		mcsClient, err := mcsclient.NewForConfig(restConfig)
 		if err != nil {
-			return err
+			accumulatedErrors = append(accumulatedErrors, err)
+			continue
+		}
+
+		if _, err := mcsClient.MulticlusterV1alpha1().ServiceExports("").List(context.TODO(), metav1.ListOptions{}); err != nil {
+			accumulatedErrors = append(accumulatedErrors,
+				fmt.Errorf("error listing ServiceExports on context %s, is the MCS API installed? %w", name, err))
+			continue
+		}
+
+		if _, err := mcsClient.MulticlusterV1alpha1().ServiceImports("").List(context.TODO(), metav1.ListOptions{}); err != nil {
+			accumulatedErrors = append(accumulatedErrors,
+				fmt.Errorf("error listing ServiceImports on context %s, is the MCS API installed? %w", name, err))
+			continue
```

Comment:
To support multiple version in the future, we'll likely retrieve the CRD resources but this is fine for now.